ftests/009-cgget-g_flag_controller_only.py

```diff
@@ -80,14 +96,18 @@ def test(config):
     elif version == CgroupVersion.CGROUP_V2:
         expected_out = EXPECTED_OUT_V2
 
-    if len(out.splitlines()) != len(expected_out.splitlines()):
-        result = consts.TEST_FAILED
-        cause = (
-                    'Expected {} lines but received {} lines'
-                    ''.format(len(expected_out.splitlines()),
-                              len(out.splitlines()))
-                )
-        return result, cause
+    if (len(out.splitlines()) != len(expected_out.splitlines()) and
```

Comment:
Will this catch the case where a cgroup v1 system has a different number of lines?

Reply:
@drakenclimber Thanks for the review.  Agree, it will not. Let me try and re-spin V3, addressing it.